foundations/introduction/motivation_and_mindset.md

```diff
@@ -156,6 +156,12 @@ Students often compare themselves to others who are farther along in their codin
 
 **Solution:** Only compare yourself to your past self. Have your abilities and knowledge improved from where you were last week, last month, or last year? Be proud of the progress that you've made!
 
+#### Counter Productive Note-Taking
```

Comment:
```suggestion
#### Counter productive note-taking
```

Someone else did a PR for this lesson today. Must have passed each other. You can click `commit suggestion` if you like so you don't have to locally edit this one.